README.md

```diff
@@ -16,34 +16,48 @@ the packages in your project and update them when a stable tag is out.
 
 [About ARM](/arm/README.md)
 
+- [account] (/arm/datalake-analytics/account)
+            (/arm/datalake-store/account)
+- [analysisservices](/arm/analysisservices)
 - [authorization](/arm/authorization)
 - [batch](/arm/batch)
 - [cdn](/arm/cdn)
 - [cognitiveservices](/arm/cognitiveservices)
+- [commerce](/arm/commerce)
+- [commitmentplans](/arm/machinelearning/commitmentplans)
```

Comment:
For this packages that are inside other packages, what do you think about including the service they belong to?